Pod/Core/XCTestCase+Gherkin.swift

```diff
@@ -17,7 +17,7 @@ UITestCase - a subclass wouldn't work with both of them.
 It's nicer code IMHO to have the state as a single associated property beacuse of the grossness of setting/getting it.
 This means that anytime I want to access my extra properties I just do `state.{propertyName}`
 */
-private class GherkinState {
+public class GherkinState {
```

Comment:
yes that's true, we don't need public, I changed it. The GherkinState is only accessed within the XCTest-Gherkin project itself.